docs/operations-guide/start.md

```diff
@@ -15,6 +15,7 @@
 *  [Configuring Logging Level](#configuring-logging-level)
 *  [How to setup monitoring via JMX](#monitoring-via-jmx)
 *  [A word on Java versions](#java-versions)
+*  [Connecting Metabase to databases in your organization](../administration-guide/01-managing-databases.md)
```

Comment:
Maybe this item should go higher up in the list since it happens earlier in the process of getting Metabase up and running.